_release_notes/armoryspinnaker_v2.18.1.md

```diff
@@ -0,0 +1,211 @@
+
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+
+## Known Issues
+There are currently no known issues with this release.
+
+<!--- Example of a problem
+Igor added ..... which does.....
+
+**Symptoms:**
+**Fix:**
+-->
+
+
+
+## Highlighted Updates
+### Armory
+This release includes the following:
+  - Improved error handling in Pipelines as Code and its microservice, Dinghy.
+  - The Terraform integration now uses Clouddriver's built-in support for `git/repo` artifact types. Previously, you had to create a custom artifact that was a `git/repo` type.
+  - Improve edgecase handling in Policy Engine; support more complex policies that inclue other top-level declarations besides `deny`.
+  - Addresses a number of CVE's related to `springframework`, `tomcat`, and `jackson-databind` found in Spinnaker Java services
+
+
+
+###  Spinnaker Community Contributions
+REMOVE ME: link to the spninaker release: https://www.spinnaker.io/community/releases/versions
+REMOVE ME: maybe copy their notable changes here
+
+<!--- Example message
+There have also been numerous enhancements, fixes and features across all of Spinnaker's other services. See their changes here:  
+[Spinnaker's v1.7.0](https://www.spinnaker.io/community/releases/versions/1-7-0-changelog)  
+[Spinnaker's v1.7.1](https://www.spinnaker.io/community/releases/versions/1-7-1-changelog)  
+-->
+
+<br>
+
+## Detailed Updates
+
+### Bill of Materials
+Here's the bom for this version.
+<details><summary>Expand</summary>
+<pre class="highlight">
+<code>version: 2.18.1-rc1286
+timestamp: "2020-04-03 15:09:43"
+services:
+  clouddriver:
+    version: 6.5.6-40c9a8c-6a11468-rc30
+  deck:
+    version: 2.15.1-367a2b6-b4e03e7-rc13
+  dinghy:
+    version: 0.0.4-16cebe7-rc1198
+  echo:
+    version: 2.10.2-925793f-ee73077-rc14
+  fiat:
+    version: 1.9.0-6bc50c7-695585e-rc10
+  front50:
+    version: 0.21.1-ed17bd1-efb3326-rc13
+  gate:
+    version: 1.14.0-23b4000-86b6626-rc15
+  igor:
+    version: 1.8.2-2066eb2-a6fa3a7-rc12
+  kayenta:
+    version: 0.13.0-c02d436-dcad431-rc10
+  monitoring-daemon:
+    version: 0.16.1-7d506f0-rc2
+  monitoring-third-party:
+    version: 0.16.1-7d506f0-rc2
+  orca:
+    version: 2.12.2-a0c169e-5a77dcb-rc18
+  rosco:
+    version: 0.17.0-8f59952-f8e1d2c-rc11
+  terraformer:
+    version: 0.0.2-4ed31a9-rc18
+dependencies:
+  redis:
+    version: 2:2.8.4-2
+artifactSources:
+  dockerRegistry: docker.io/armory</code>
+</pre>
+</details>
+
+
+
+### Armory
+#### Dinghy&trade; - bb15163...16cebe7
+ - fix(notif): surface policy engine errors in notifications (#194)
+
+#### Terraformer&trade; - 266ac86...4ed31a9
+ - feature(profiles): support git-ssh profiles (#121)
```

Comment:
questionable whether we should call out named profiles yet. @brian-armory

Reply:
Is it in a stable state where it can be used relatively bug-fee? If yes, let's call it out in the highlights section.

Reply:
We should merge the PR for the docs for Named Profiles too.

Reply:
Let's punt to Beth ;) If she says yes, this is likely what'd we want to add to the highlights section.

```  - The Terraform Integration now supports `git-ssh` profiles now. A profile gives users the ability to reference certain kinds of external sources, such as a private remote repository, when creating pipelines. For more information, see the [Terraform Integration](/spinnaker/terraform_integration) documentation.```

Reply:
resolved to not surface it until it's better tested